README.md

```diff
@@ -1,4 +1,6 @@
-# smimesign (S/MIME Sign) [![Windows Build Status](https://ci.appveyor.com/api/projects/status/hs3o1m9ornnp9d52/branch/master?svg=true)](https://ci.appveyor.com/project/github/smimesign-0bhgj/branch/main) [![macOS Build Status](https://travis-ci.org/github/smimesign.svg?branch=main)](https://travis-ci.org/github/smimesign)
+# smimesign (S/MIME Sign)
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/github/smimesign?utm_source=godoc)](https://pkg.go.dev/github.com/github/smimesign?tab=doc) [![Windows Build Status](https://ci.appveyor.com/api/projects/status/hs3o1m9ornnp9d52/branch/master?svg=true)](https://ci.appveyor.com/project/github/smimesign-0bhgj/branch/main) [![macOS Build Status](https://travis-ci.org/github/smimesign.svg?branch=main)](https://travis-ci.org/github/smimesign)
```

Comment:
I think while we are here, we can get rid of these AppVeyor and Travis badges since we are on Actions entirely now?